db/migrate/20250616095102_add_reminder_mails_active.rb

```diff
@@ -0,0 +1,8 @@
+class AddReminderMailsActive < ActiveRecord::Migration[8.0]
+  def change
+    add_column :people_skills, :created_at, :datetime
```

Comment:
https://guides.rubyonrails.org/v6.1/active_record_migrations.html#migration-overview
No need to do that by hand